source/usage-examples/deleteOne.txt

```diff
@@ -3,3 +3,51 @@ Delete a Document
 =================
 
 .. default-domain:: mongodb
+
+You can delete a single document in a collection using the
+``DeleteOne()`` method.
+
+The following example specifies a query filter that matches documents in
+the ``movies`` collection with the value "Twilight" in the ``title`` field, and
+deletes the first document that matches. 
```

Comment:
```suggestion
The following example deletes a document in
the ``movies`` collection of the ``sample_mflix`` database with the value "Twilight" in the ``title`` field.
```

Suggestion: I think "specify a query filter that matches" can be simplified to "deletes". I think the more detailed explanation should be included in the delete fundamentals page. 

Suggestion: I think if the collection is mentioned it makes sense to also mention the database.

Reply:
I didn't implement this change because it strays from the template we agreed on as a team, but I'm open to having a team discussion on how we should word the example statement.

Reply:
Zoom call. Determined that we have different interpretations of the usage example template. Should bring this up in a team discussion later on.